### DISCUSSION TITLE
European date format

### DISCUSSION THREAD

**Discussion by johnwuelk** [jarun/nnn#1710] [Ideas]
Good day everyone,

Is it possible to configure or switch to a European date format like DD.MM.YYYY?  (in detail mode)

The American way that the month comes before the day is overly confusing to me.

Thanks for the awesome file manager!

**Comment by jarun** [comment]
It's the same here. In India the the day comes before the month.
However, this is per the ISO 8601 standard. No plans to change.

**Comment by johnwuelk** [comment]
Fair enough.
